### ISSUE TITLE
useTransformContext() - Transform context must be placed inside TransformWrapper

### ISSUE THREAD

**Issue by tandrasi** [prc5/react-zoom-pan-pinch#352]
**Describe the bug**

When following the new hooks in version 3.0.2:

```
// Simple access to the context instance
const context = useTransformContext();
```

I keep on getting an error that says:

`Transform context mus be placed inside TransformWrapper` (notice small "must" typo in the code too)

**To Reproduce**
put the following at top of app, inside the TransformWrapper:
const context = useTransformContext();

**Expected behavior**
As per documentation, I should be able to access the transform context, so I can access **scale**.

**Additional context**
Have been using component for a while now building this new project. Older versions work great as I was using "canvasRef.current.state.scale". I wanted the latest and greatest due to performance enhancements via new hooks.

**Comment by prc5** [comment]
Hey! It clearly mean that your context was lost. This hook needs to be placed inside another component which is used in the TransformWrapper. If previous approach with canvasRef worked for you, you can still use it to access state outside 👍

**Comment by tandrasi** [comment]
Ok, that's a bit clearer. My issue was that upon installing 3.02, I was under the impression I'd need to do a few updates due to breaking changes so when I booted my project up, **the following no longer worked and led to run-time errors** (even though it was allowed in the typings):

```
**myRef**.current.state.scale
**myRef**.current.state.positionX === 0
**myRef**.current.state.positionY === 0
```


So I immediately started to implement the useTransformContext() as I thought I needed to use it in place of the ReactZoomPanPinchRef. After your message, I found the correct properties eventually:

```
**myRef**.current.instance.transformState.scale === 1
**myRef**.current.instance.transformState.positionX === 0
**myRef**.current.instance.transformState.positionY === 0
```

Thanks for the hard work on this repo!
